amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/table/TableOptimizingMetrics.java

```diff
@@ -163,6 +165,37 @@ public class TableOptimizingMetrics {
           .withTags("catalog", "database", "table")
           .build();
 
+  public static final MetricDefine TABLE_LAST_MINOR_OPTIMIZING_INTERVAL =
+      defineGauge("table_last_minor_optimizing_duration_mills")
```

Comment:
Considering that the metrics related to optimizing on the table are currently prefixed with `table_optimizing`, I suggest renaming this monitoring metric to `table_optimizing_minor_time_mills_since_last_completion` or `table_optimizing_minor_duration_mills_since_last_completion`.